### PR TITLE
made component for itemlist and markup for listing page

### PR DESCRIPTION
**Pull request by johnstonstu**
made a component to display the item list from the db and some basic markup for the listing page, search and filter functionality still tbd

### CODE REVIEW

File: js/screens/Listings/Listings.js [hunk 1]
```diff
@@ -1,9 +1,32 @@
 import React from 'react';
-import { Text, View } from 'react-native';
-const Listings = () => {
+import { Text, View, TextInput, Image, TouchableOpacity } from 'react-native';
+import ItemList from '../../components/ItemsList';
+import styles from './styles';
+import { colors } from '../../config/styles';
+
+const Listings = ({ data }) => {
+  console.log(data);
```
Comment (siddharthparmar7) [inline js/screens/Listings/Listings.js]: Please remove this console log.

File: js/screens/Listings/Listings.js [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
-      <Text>Listings Screen</Text>
+    <View styles={styles.container}>
+      <View style={styles.searchContainer}>
+        <Image
+          style={styles.searchIcon}
+          source={require('../../assets/images/Icons/Search.png')}
+        />
+        <TextInput
+          style={styles.searchInput}
+          onChangeText={text => console.log(text)}
```
Comment (siddharthparmar7) [inline js/screens/Listings/Listings.js:17]: Please remove this console log

Comment (siddharthparmar7) [reply js/screens/Listings/Listings.js:17]: is this a placeholder?

File: js/screens/Listings/Listings.js [hunk 3]
```diff
@@ -1,9 +1,32 @@
...
+          placeholder={'Enter a Keyword or Location'}
+          placeholderTextColor="#0082B566"
+        />
+        <TouchableOpacity onPress={() => console.log('clicked filter')}>
```
Comment (siddharthparmar7) [inline js/screens/Listings/Listings.js:21]: placeholder?

File: js/screens/Listings/ListingsContainer.js [hunk 4]
```diff
@@ -1,10 +1,42 @@
 import React, { Component } from 'react';
 import Listings from './Listings';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import LoadingIndicator from '../../components/LoadingIndicator';
+const GET_ITEMS = gql`
```
Comment (siddharthparmar7) [inline js/screens/Listings/ListingsContainer.js]: Your gql queries should be in its relative file. config/queries.js

File: js/screens/Listings/styles.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    paddingHorizontal: width / 8,
+    paddingVertical: 8,
+    width: width / 1.3,
+  },
+  searchIcon: {
+    position: 'absolute',
+    top: 11,
+    left: 20,
+    height: 20,
+    width: 20,
```
Comment (siddharthparmar7) [inline js/screens/Listings/styles.js]: is there a better way to do this? can we try? if you need help let me know.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the requested changes. Thank you!

File: js/screens/Listings/Listings.js (hunk 3 above)
Comment (johnstonstu) [reply js/screens/Listings/Listings.js:21]: yes

File: js/screens/Listings/Listings.js (hunk 2 above)
Comment (johnstonstu) [reply js/screens/Listings/Listings.js:17]: yes

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**